src/main/java/com/bootcamp/demo/DemoApplication.java

```diff
@@ -7,7 +7,12 @@
 public class DemoApplication {
 
     public static void main(String[] args) {
-        SpringApplication.run(DemoApplication.class, args);
+        //trying to catch the error from the Card constructor
+        try {
+            SpringApplication.run(DemoApplication.class, args);
+        } catch (Exception e){
+            System.out.println(e.getMessage());
+        }
+        //SpringApplication.run(DemoApplication.class, args);
```

Comment:
Please don't modify the application's main method. Exception handling should be implemented at the Service/Controller level.